run_development.sh

```diff
@@ -1,15 +1,18 @@
 #!/bin/bash
 
 CURRENT_PATH="$(cd "$(dirname "$1")"; pwd)/$(basename "$1")"
- 
+
+docker pull ghcr.io/navikt/pdfgen:2.0.30
```

Comment:
Må versjonen hardkodes her? Den vil vel veldig fort komme i usync med det som ligger i Dockerfile, som bumpes av Dependabot. For meg fungerte den å kjøre as is

Reply:
Nei, tenkte da slapp vi å lage en egen syfopdfgen image lokalt. Men jeg reverter